src/task/script.js

```diff
@@ -1,4 +1,4 @@
-const BASE_URL = 'https://api.realdevsquad.com';
+const BASE_URL = window.BASE_URL;
```

Comment:
BASE URL for what? What if we call it `API_BASE_URL`?

Reply:
Right 👍